src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DSA/DSASignVerify.cs

```diff
@@ -163,7 +163,17 @@ private void UseAfterDispose(bool importKey)
             Assert.Throws<ObjectDisposedException>(
                 () =>
                 {
-                    key.KeySize = 576;
+                    try
+                    {
+                        key.KeySize = 576;
+                    }
+                    catch (CryptographicException) when (PlatformDetection.IsAndroid)
```

Comment:
Exception filters are problematic in AOT scenarios.  I would move this check inside the catch block.

Reply:
Do we AOT our tests?

Reply:
Yes, we send all assemblies in the publish directory, for example, to the AOT compiler.

Reply:
Huh, TIL.  Cool :smile:.